database/management/createTables.sql

```diff
@@ -52,15 +50,23 @@ CREATE TABLE IF NOT EXISTS sensors (
 CREATE TRIGGER updated_at_sensors BEFORE UPDATE ON sensors FOR EACH ROW EXECUTE PROCEDURE update_updated_at_column();
 
 
-CREATE TABLE IF NOT EXISTS measurements (
+CREATE TABLE IF NOT EXISTS outputs (
     id                  SERIAL PRIMARY KEY,
-    sensor_sim          text REFERENCES sensors (sim) NOT NULL,
+    sensor_id           integer REFERENCES sensors (id) ON DELETE CASCADE NOT NULL,
     type                text NOT NULL, -- This can be signal_strength, temperature, pressure, ... but we can't presuppose it in an enum.
-    "value"             real[] NOT NULL, -- This was changed from signal_strengths to measurements
+    status              text DEFAULT 'NODATA'
+) INHERITS(lifecycle);
+CREATE TRIGGER updated_at_outputs BEFORE UPDATE ON outputs FOR EACH ROW EXECUTE PROCEDURE update_updated_at_column();
+
+
+CREATE TABLE IF NOT EXISTS measurements (
+    id                  SERIAL PRIMARY KEY,
+    output              integer REFERENCES outputs (id) NOT NULL,
```

Comment:
This is inconsistent with `sensor_id` above. No strong opinion on either, but let's be consistent.